Heatington/Optimizer/Optimizer.cs

```diff
@@ -105,10 +101,10 @@ int SatisfyHeatDemand(DataPoint dataPoint)
 
             while (dataPoint.HeatDemand > currentProductionCapacity)
```

Comment:
I wrote a for loop for this, but it got lost in some merges

A while loop also works